public/static/docs/command-reference/update.md

```diff
@@ -30,10 +30,10 @@ update them.
 Another detail to note is that when the `--rev` (revision) option of
 `dvc import` has been used to create an import stage, DVC is not aware of what
 kind of
-[Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References) this
-is, for example a branch or a tag. For typically static references (e.g. tags),
-or for SHA commits, `dvc update` will not have any effect on the import. Refer
-to the
+[Git reference](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+this is, for example a branch or a tag. For typically static references (e.g.
+tags), or for commits hashes, `dvc update` will not have any effect on the
```

Comment:
`commit hashes` ... probably better to do `commit SHA hashes` then

Reply:
Yeah we have `commit hash`/`commit hashes` all over the docs too so I went for that instead of `SHA` but you're probably right. I'll review all instances again...

Reply:
How about just linking "hash" to https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefhashahash ? And try avoiding "SHA".

Reply:
it's about "pattern matching" when people read. A lot of professionals (and even beginners) very familiar with `commit sha` term/phrase. While I can understand that some folks don't care about it being `sha`, and I understand that it looks cleaner and nicer and easier to support, etc - we should still think about how general audience reads this, and make it as faster to understand as possible first, about cleanliness and other metrics second.

Reply:
Got it. Updated in f10cf2b2. Another huge commit, sorry 😅